diff -rupN mpi4py-2.0.0.orig/setup.cfg mpi4py-2.0.0/setup.cfg
--- mpi4py-2.0.0.orig/setup.cfg	2016-09-14 10:12:14.000000000 -0400
+++ mpi4py-2.0.0/setup.cfg	2016-09-14 10:12:32.000000000 -0400
@@ -17,3 +17,5 @@ tag_build = 
 tag_date = 0
 tag_svn_revision = 0
 
+[build_ext]
+define = PyMPI_MISSING_MPI_Type_create_f90_integer, PyMPI_MISSING_MPI_Type_create_f90_real, PyMPI_MISSING_MPI_Type_create_f90_complex
