diff -rupN mpi4py-1.3.1.orig/setup.cfg mpi4py-1.3.1/setup.cfg
--- mpi4py-1.3.1.orig/setup.cfg	2012-01-20 13:38:42.000000000 -0500
+++ mpi4py-1.3.1/setup.cfg	2015-09-22 17:22:21.002460259 -0400
@@ -26,3 +26,6 @@ group     = Libraries/Python
 doc_files = README.txt HISTORY.txt LICENSE.txt THANKS.txt
 #           docs/index.html docs/mpi4py.pdf
 #           docs/usrman docs/apiref
+
+[build_ext]
+define = PyMPI_MISSING_MPI_Type_create_f90_integer, PyMPI_MISSING_MPI_Type_create_f90_real, PyMPI_MISSING_MPI_Type_create_f90_complex
